.travis.yml

```diff
@@ -34,3 +34,9 @@ script:
         source ./.CI/build_all;
       fi
 
+after_failure:
+  # Trigger a build only if this is not a pull request and is on the `master` branch.
+  # Also ensure that no recipes were converted (otherwise we already triggered a build by pushing).
+  - ( [ "${TRAVIS_PULL_REQUEST}" == "false" ] && [ "${TRAVIS_BRANCH}" == "master" ] ) && \
```

Comment:
You also need to check that this is `conda-forge` and not a fork. (Easiest would be to check for `$GH_TOKEN`)

Reply:
Good point. Added a check for the slug. Though presumably the Python script will fail in some ugly way if this not the right repo.